container/common/helpers.go

```diff
@@ -224,6 +251,31 @@ func readString(dirpath string, file string) string {
 	return strings.TrimSpace(string(out))
 }
 
+// Convert from [1-10000] to [2-262144]
+func convertCPUWeightToCPULimit(weight uint64) (uint64, error) {
+	if weight < 1 || weight > 10000 {
```

Comment:
Can you introduce constants instead of using magic numbers, please? It would help me to understand what these two values represent.

Reply:
sure, going to clean it up